src/applications/facility-locator/components/ResultsList.jsx

```diff
@@ -99,7 +99,7 @@ class ResultsList extends Component {
         {/* eslint-disable jsx-a11y/no-noninteractive-tabindex */}
         <p className="search-result-title" ref={this.searchResultTitle}>
           {/* eslint-enable jsx-a11y/no-noninteractive-tabindex */}
-          {`${results.length} results for ${facilityTypeName} near `}
+          {`${totalEntries} results for ${facilityTypeName} near `}
```

Comment:
`totalEntries` is the complete count of search results, whereas `results.length` is the count of the current page. `totalEntries` makes more sense.